Documentation/building/cross-platform-testing.md

```diff
@@ -24,13 +24,15 @@ instructions assume you are building for Linux, but are easily modifiable for OS
    respect to object sizes and layout so you need to ensure you have either a
    release coreclr and release mscorlib or debug coreclr and debug mscorlib.
 3. A Linux build of CoreFX. We currently have experimental support for building
-   CoreFX on Linux via `build.sh`. The other option is to build the managed
-   parts of CoreFX on Windows, To do so run `build.cmd /p:TargetOS=Linux
-   /p:OSGroup=Linux`. It is okay to build a Debug version of CoreFX and run it
+   CoreFX on Linux via `build.sh`. 
+   The other option is:
+
+   * Build the managed parts of CoreFX on Windows. To do so run `build-managed.cmd -os=Linux -target-os=Linux`. It is okay to build a Debug version of CoreFX and run it
```

Comment:
To be honest I'm also a little fuzzy on all the different options. We really do need to go through all the options in our config.json file and get the properly documented. @maririos this is what I was hinting about with needing to iterate on the config file, would you mind filing some issues to get these properties better documented both in the config file as well as the documentation files?